cosmos-init/plugins/packages/cosmosc2-tool-common/src/services/cable.js

```diff
@@ -17,7 +17,9 @@
 # copyright holder
 */
 
-import * as ActionCable from 'actioncable'
+import * as ActionCable from '@rails/actioncable'
+//ActionCable.logger.enabled = true
+ActionCable.ConnectionMonitor.staleThreshold = 60
```

Comment:
Looks like the [default](https://github.com/rails/rails/blob/ed8e7672180aff0101da0826087bf4ca07094f02/actioncable/app/assets/javascripts/actioncable.js#L111) is 6. Do you think this is what did it?